org.lflang/src/org/lflang/generator/LFGenerator.java

```diff
@@ -100,6 +101,43 @@ private FileConfig createCppFileConfig(Resource resource,
         }
     }
 
+    /**
+     * Create a TypeScript specific FileConfig object
+     *
+     * Since the CppFileConfig class is implemented in Kotlin, the class is is
```

Comment:
The comments still reference `CppFileConfig`

Reply:
Thanks, updated the comments.